modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase

```diff
@@ -245,6 +245,45 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test ExportCustomObjectWithoutObjectEntry {
```

Comment:
Looks good, thanks @pedroamarall . The only thing is there are some inconsistent between the name of the testcase "ExportCustomObjectWithoutObjectEntry" and the code in Line 258. I mean in the name we use withoutObjectEntry, but what we tested in the tests Line258, we are using one Student entry "Jessica", could you confirm which is correct and improve accordingly? Thanks! :smile:

Reply:
Hi @yanan , thanks for the feedback, it really doesn't make sense, I removed the "add Object Entry" and did a push

Reply:
@pedroamarall No worries, I think we should also improve the assert in Line 270 since there is no "Jessica" anymore once we removed "add Object Entry".